sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py

```diff
@@ -187,12 +187,14 @@ def send(self, events, **kwargs):
         :keyword str content_type: The type of content to be used to send the events.
          Has default value "application/json; charset=utf-8" for EventGridEvents,
          with "cloudevents-batch+json" for CloudEvents
+        :keyword str channel_name: Optional. Required only when publishing to partner namespaces with partner topic
```

Comment:
We could debate if the main signature should be `send(self, events, * channel_name=None, **kwargs)`

cc @johanste

Reply:
This feature will have very low usage (les than 1%) - i don't believe it's worth putting it in the method signature

Reply:
If we add the documentation, then we may as well make it keyword-only (as opposed to a `kwarg`). I'm still not entirely sure exactly who can make use of the feature. If it is truly 1p and a very limited number of 3p that will never grow that can make use of it, then adding it to the public surface area for everyone probably doesn't make sense. 

We pick one side of the fence. Document + keyword only, or don't document and kwarg.

Reply:
"If it is truly 1p and a very limited number of 3p" - this is not entirely true. Anyone can become a partner and is not restricted to 1p.

There is absolutely no reason to not document it - let's do document  + keyword only